### PR TITLE
little improvement on start page

### PR DESCRIPTION
**Pull request by luizbills**
### before:

before
### after:

after

### CODE REVIEW

**Comment by luizbills** [comment]
fixed

File: app/assets/stylesheets/app/components/lists.scss [hunk 1]
```diff
@@ -103,7 +103,9 @@
 
     > .body {
       margin-top: -145px;
-      width: 100%
+      width: 100%;
```
Comment (luizbills) [inline app/assets/stylesheets/app/components/lists.scss]: fixed

**Comment by lachlanjc** [comment]
@luizbills It would be better to use the `.px1` or `.px2` helper classes straight on the HTML instead of adding more CSS, but this would be okay. We're generally trying to remove as much CSS as possible ATM, because it's 381KB :)

**Comment by luizbills** [comment]
@lachlanjc That's true. `.px2` will be better.

**Comment by lachlanjc** [comment]
:+1:

**Pull request closed without merging**